src/tests/Common/Directory.Build.targets

```diff
@@ -103,22 +103,20 @@
           Include="$(CoreCLRArtifactsPath)%(RunTimeArtifactsIncludeFolders.Identity)**/*"
           Exclude="@(RunTimeArtifactsExcludeFiles -> '$(CoreCLRArtifactsPath)%(Identity)')"
           TargetDir="%(RunTimeArtifactsIncludeFolders.Identity)" />
+    </ItemGroup>
 
-      <RunTimeDependencyCopyLocal
-          Condition="'$(TargetArchitecture)' != 'x64'"
-          Include="$(CoreCLRArtifactsPath)x64/crossgen2/clrjit_*"
-          TargetDir="crossgen2/" />
-
-      <RunTimeDependencyCopyLocal
-          Condition="'$(TargetArchitecture)' != 'x64'"
-          Include="$(CoreCLRArtifactsPath)x64/crossgen2/jitinterface_*"
-          TargetDir="crossgen2/" />
+    <PropertyGroup>
+      <Crossgen2Dir />
+      <Crossgen2Dir Condition="'$(TargetArchitecture)' != 'x64' and '$(BuildArchitecture)' == 'x64'">$(CoreCLRArtifactsPath)x64/crossgen2</Crossgen2Dir>
+    </PropertyGroup>
 
+    <ItemGroup Condition="'$(Crossgen2Dir)' != ''">
       <RunTimeDependencyCopyLocal
-          Condition="'$(TargetArchitecture)' != 'x64' and '$(TargetOS)' == 'windows'"
-          Include="$(CoreCLRArtifactsPath)x64/crossgen2/Microsoft.DiaSymReader.Native.amd64.dll"
+          Include="
+            $(Crossgen2Dir)/clrjit_*;
+            $(Crossgen2Dir)/jitinterface_*;
+            $(Crossgen2Dir)/Microsoft.DiaSymReader.Native.*.dll"
```

Comment:
Using the `*` wildcard allows to omit the `'$(TargetOS)' == 'windows'"` condition — this will be resolved to the empty file set on Linux/macOS.